<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>instruction_ui</class>
 <widget class="QWidget" name="instruction_ui">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>43</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777212</width>
    <height>16777212</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_delInstruction">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="instructionText">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="status_txt">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QDateTimeEdit" name="horodate"/>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="btn_paste">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Coller</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="btn_edit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btn_validate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_delInstruction</sender>
   <signal>clicked()</signal>
   <receiver>instruction_ui</receiver>
   <slot>delInstruction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_paste</sender>
   <signal>clicked()</signal>
   <receiver>instruction_ui</receiver>
   <slot>pastePOI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_edit</sender>
   <signal>clicked()</signal>
   <receiver>instruction_ui</receiver>
   <slot>editInstruction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_validate</sender>
   <signal>clicked()</signal>
   <receiver>instruction_ui</receiver>
   <slot>validateModif()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horodate</sender>
   <signal>dateTimeChanged(QDateTime)</signal>
   <receiver>instruction_ui</receiver>
   <slot>dateTime_changed(QDateTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>instruction_ui</receiver>
   <slot>maintenant()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>instruction_ui</receiver>
   <slot>copyPOI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delInstruction()</slot>
  <slot>pastePOI()</slot>
  <slot>editInstruction()</slot>
  <slot>validateModif()</slot>
  <slot>dateTime_changed(QDateTime)</slot>
  <slot>maintenant()</slot>
  <slot>copyPOI()</slot>
 </slots>
</ui>
